src/main/java/spongebob/Parser.java

```diff
@@ -0,0 +1,91 @@
+package spongebob;
+
+import spongebob.command.AddCommand;
+import spongebob.command.Command;
+import spongebob.command.DeleteCommand;
+import spongebob.command.DisplayCommand;
+import spongebob.command.ExitCommand;
+import spongebob.command.FindCommand;
+import spongebob.command.MarkCommand;
+import spongebob.command.UnknownCommand;
+/**
+ * object to parse user input into commands that can be executed by Spongebob
+ */
+public class Parser {
+
+    private String[] arguments;
+
+    /**
+     * the command syntax is a array of size 4,
+     * first string will be the command,
+     * second is description,
+     * third is from / by date,
+     * last is the to date
+     */
+    public Parser() {
+        this.arguments = new String[4];
+
+        for (int i = 0; i < arguments.length; i++) {
+            this.arguments[i] = " ";
+        }
+    }
+
+    /**
+     * Parses command into a Command object to be executed
+     * @param string User input for commands
+     * @return A executable command
+     */
+    public Command parse(String string) {
+
+        String[] command = string.split(" ", 2);
```

Comment:
Since this variable will store multiple values as a string array, I suggest changing the name `command` to `commands`. What do you think?

Reply:
Yes, you are correct, that makes more sense!